cardano-cli/test/cardano-cli-golden/files/golden/help/latest_genesis_create-staked.cli

```diff
@@ -31,7 +31,7 @@ Available options:
                            [default is 0].
   --gen-stake-delegs INT   The number of stake delegator credential sets to make
                            [default is 0].
-  --start-time UTC-TIME    The genesis start time in YYYY-MM-DDThh:mm:ssZ
+  --start-time UTC_TIME    The genesis start time in YYYY-MM-DDThh:mm:ssZ
```

Comment:
Remove this change from this PR and open an ADR for it.

Reply:
Removed it from this commit: https://github.com/IntersectMBO/cardano-cli/commit/513622cc826f0f7cce3e90ec8a57af83f797368e

Reply:
Created issue here: https://github.com/IntersectMBO/cardano-cli/issues/1148